aiounifi/interfaces/connectivity.py

```diff
@@ -74,6 +74,10 @@ async def login(self) -> None:
                 LOGGER.error("Login failed '%s'", data)
                 raise ERRORS.get(data["meta"]["msg"], AiounifiException)
 
+        elif response.content_type == "text/html":
```

Comment:
I guess we could just do an else here. But let's keep it like this for now

Reply:
I started with an else here, and the tests failed - I don't recall which one, but we do need to look for text/html - I think there is a valid text/json header in the tests.

Reply:
Some old tests are not correct so there are room for improvement, maybe Ill add a warning in an else and see if someone reports it :)

Reply:
I think it needs a bit more digging.  There are 2 if statements - else on the first one: 
https://github.com/Kane610/aiounifi/blob/8afb578803d4462fbb1ba0a635437283ccfcc70b/aiounifi/interfaces/connectivity.py#L71 
caused use to miss the second test: 
https://github.com/Kane610/aiounifi/blob/8afb578803d4462fbb1ba0a635437283ccfcc70b/aiounifi/interfaces/connectivity.py#L81
Else on the second test was not good because the response is `HTTPStatus.OK`, just the wrong content_type.
If we move the second test around - or know that the `x-crf-token` is never found in a text/html or text/json.

Reviewing one "source" https://stackoverflow.com/questions/23714383/what-are-all-the-possible-values-for-http-content-type-header it appears text/json is not even a valid content type?  If that's so, then we should only look for the `x-crf-token` in `application/json` and use an else.

Sometimes it amazes me how much thought can go into a couple of `if` statements!

Reply:
I did a quick PR https://github.com/Kane610/aiounifi/pull/641 based on what I think you were thinking :)